src/modules/Verifications/components/TypeRadioGroup/useTypeRadioGroup.tsx

```diff
@@ -0,0 +1,28 @@
+import { verificationType } from 'src/core/adaptors';
+
+export const useTypeRadioGroup = () => {
+  const cardItems = [
+    {
+      value: 'reusable' as verificationType,
```

Comment:
Don't we have translation for these card items?

Reply:
I should Add, thanks for mentioning